### PR TITLE
Homebrew 1.0.0

### PR DESCRIPTION
**Pull request by MikeMcQuaid**
- Redesign the site (with some of the worst CSS changes of all time)
- Add 1.0.0 announcement post

CC @Homebrew/maintainers for final review(s)

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -19,7 +19,8 @@
     what: The script explains what it will do and then pauses before it does it. There are more installation options <a href='https://github.com/Homebrew/brew/blob/master/docs/Installation.md#installation'>here</a> (needed on 10.5).
   doc:
     further: Further Documentation
+    blog: Homebrew Blog
   foot:
-    code: Original code by <a href="https://mxcl.github.io/">Max Howell</a>.
-    page: Website by <a href="http://exomel.com">Rémi Prévost</a>.
+    code: Homebrew was originally created by <a href="https://mxcl.github.io/">Max Howell</a>.
```
Comment (dunn) [inline index.html]: This can just be "was created by".

File: index_da.html [hunk 2]
```diff
@@ -20,8 +20,9 @@
     paste: Kopier og indsæt i din Terminal
     what: Scriptet forklarer hvad det vil gøre, og pauser før den gør det. Der findes flere installeringsmuligheder <a href='https://github.com/Homebrew/brew/blob/master/docs/Installation.md#installation' title="Flere installeringsmuligheder">her</a> (krævet i 10.5).
     doc:
-    further: Videre dokumentation
+    further: Further Documentation
```
Comment (dunn) [inline index_da.html]: Why switch all these to English?

Comment (MikeMcQuaid) [inline index_da.html]: Nice spot; regex search and replace gone wrong.

File: index.html (hunk 1 above)
Comment (MikeMcQuaid) [inline index.html]: 👍

File: _posts/2016-09-02-homebrew-1.0.0.md [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+Today I'm proud to announce the release of Homebrew 1.0.0. We've been working hard over the last year to make some major changes to Homebrew that we've been wanting for a long time. There's been some hiccups along the way but we now have a more stable base for using and developing Homebrew in future.
+
+These include:
+
+- [Homebrew package management and formulae (package metadata) split into separate repositories](https://github.com/Homebrew/brew/pull/2)
+- [Homebrew Community site (using Discourse) for discussion](http://community.brew.sh)
+- [Homebrew CI and homebrew/core use the OS X sandbox for build-time security](https://github.com/Homebrew/legacy-homebrew/pull/38361)
+- [Homebrew Cask integrated into Homebrew/brew](https://github.com/Homebrew/brew/pull/725)
+- [Add new `brew boneyard-formula-pr` command to create new formula version pull requests](https://github.com/Homebrew/brew/pull/53)
+- [Add `brew --help` to `brew` subcomands](https://github.com/Homebrew/brew/pull/10)
```
Comment (bfontaine) [inline _posts/2016-09-02-homebrew-1.0.0.md]: Typo here: “subcomands” → “subcommands”

File: _posts/2016-09-02-homebrew-1.0.0.md [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+- Officially support [`brew bundle`](https://github.com/Homebrew/homebrew-bundle) (for `Brewfile`s and import/export) and [`brew services`](https://github.com/Homebrew/homebrew-services) (for background services management).
+- [Homebrew/brew passes tests on Linux and has generic backend for porting to other platforms in future](https://github.com/Homebrew/brew/pull/168)
+- [Provide access to developer commands and `brew update` workflow automatically](https://github.com/Homebrew/brew/pull/881)
+- [Use `curl` for all HTTP access for consistent proxy support](https://github.com/Homebrew/brew/pull/295)
+- Use [new Ruby Macho library](https://github.com/Homebrew/ruby-macho) for reading and writing library macOS Mach O file locations.
+- [Provide vendored, portable Ruby 2 binary for when system Ruby 2 is not available](https://github.com/Homebrew/brew/pull/404)
+- [`HEAD` package installations have versions and can be upgraded](https://github.com/Homebrew/brew/pull/584)
+- [Use Python virtualenvs to handle Python dependencies better](https://github.com/Homebrew/brew/pull/344)
```
Comment (bfontaine) [inline _posts/2016-09-02-homebrew-1.0.0.md]: I’m not a native but shouldn’t that be “to better handle Python dependencies”?

Comment (zmwangx) [inline _posts/2016-09-02-homebrew-1.0.0.md]: 100% sure both forms are fine.

File: _posts/2016-09-02-homebrew-1.0.0.md [hunk 5]
```diff
@@ -0,0 +1,30 @@
+---
+title: 1.0.0
+author: MikeMcQuaid
+---
+Today I'm proud to announce the release of Homebrew 1.0.0. We've been working hard over the last year to make some major changes to Homebrew that we've been wanting for a long time. There's been some hiccups along the way but we now have a more stable base for using and developing Homebrew in future.
```
Comment (zmwangx) [inline _posts/2016-09-02-homebrew-1.0.0.md]: "There's been some hiccups" => "There have been some hiccups"

File: _posts/2016-09-02-homebrew-1.0.0.md (hunk 4 above)
Comment (DomT4) [inline _posts/2016-09-02-homebrew-1.0.0.md]: Either is fine, but Baptiste's suggestion does read a little nicer.

File: _layouts/index.html [hunk 6]
```diff
@@ -121,6 +121,15 @@ <h2 id="further-doc">{{ page.pagecontent.doc.further }}</h2>
     </li>
 
     <li>
+      <div class="group row">
+        <h2 id="homebrew-blog">{{ page.pagecontent.doc.blog }}</h2>
+        <div class="button">
+          <p><a href="/blog">https://brew.sh/blog</a></p>
```
Comment (DomT4) [inline _layouts/index.html]: This still draws a giant `Your connection is not private` for now.

File: _posts/2016-09-02-homebrew-1.0.0.md [hunk 7]
```diff
@@ -0,0 +1,30 @@
...
+
+These include:
+
+- [Homebrew package management and formulae (package metadata) split into separate repositories](https://github.com/Homebrew/brew/pull/2)
+- [Homebrew Community site (using Discourse) for discussion](http://community.brew.sh)
+- [Homebrew CI and homebrew/core use the OS X sandbox for build-time security](https://github.com/Homebrew/legacy-homebrew/pull/38361)
+- [Homebrew Cask integrated into Homebrew/brew](https://github.com/Homebrew/brew/pull/725)
+- [Add new `brew boneyard-formula-pr` command to create new formula version pull requests](https://github.com/Homebrew/brew/pull/53)
```
Comment (DomT4) [inline _posts/2016-09-02-homebrew-1.0.0.md]: I think you mislabelled this one? You've linked/referenced the boneyard command but seem to be talking about `bump-formula-pr` ❓

**Comment by zmwangx** [comment]
Can we maybe remove that godawful "Fork me on GitHub"? It doesn't fit into our color scheme for a start.

**Comment by MikeMcQuaid** [comment]
@zmwangx I think it's nice to have it but I don't mind changing the colour.

File: _posts/2016-09-02-homebrew-1.0.0.md [hunk 8]
```diff
@@ -0,0 +1,35 @@
...
+We've been working hard over the last year to make some major changes to Homebrew that we've been wanting for a long time. There have been some hiccups along the way but we now have a more stable base for using and developing Homebrew in future.
+
+These include:
+
+- [Homebrew package management and formulae (package metadata) split into separate repositories](https://github.com/Homebrew/brew/pull/2)
+- [Homebrew Community site (using Discourse) for discussion](http://community.brew.sh)
+- [Homebrew joined the Software Freedom Conservancy](https://sfconservancy.org/news/2016/feb/22/homebrew-joins/)
+- [Homebrew's default repository installation location changed to `/usr/local/Homebrew` to keep your `/usr/local` cleaner](https://github.com/Homebrew/install/pull/60)
+- [Homebrew/brew updates between release tags](https://github.com/Homebrew/brew/pull/998)
+- [Homebrew CI and homebrew/core use the OS X sandbox for build-time security](https://github.com/Homebrew/legacy-homebrew/pull/38361)
```
Comment (DomT4) [inline _posts/2016-09-02-homebrew-1.0.0.md]: In hindsight, given we're using `macOS` almost everywhere in code, we probably should have used it here as well. Not a big deal.

**Pull request merged**